health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/WorkflowCommentPopUp.js

```
@@ -119,65 +107,55 @@ const WorkflowCommentPopUp = ({ onClose, heading, submitLabel, url, requestPaylo
-                        {error && (
-                            <ErrorMessage
-                                message={t('HCM_MICROPLAN_ADD_COMMENT_REQUIRED')}
-                                truncateMessage={true}
-                                maxLength={256}
-                                showIcon={true}
-                            />
-                        )}
+        <PopUp
+            onClose={onClose}
+            heading={t(heading)}
+            children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)